PackageViewModel/PackageChooser/PackageInfoViewModel.cs

```diff
@@ -283,6 +318,11 @@ private void OnCancelDownload()
             }
         }
 
+        private void OnOpenAlternatePackage(string packageId)
+        {
+            _parentViewModel.SearchCommand.Execute("id:" + packageId);
```

Comment:
We should probably log a telemetry event for Opening an alternate package like we do the other user actions/events.

Reply:
Tracked as: `PackageInfoViewModel_OpenAlternatePackage`